torchtext/csrc/vectors.cpp

```diff
@@ -3,12 +3,19 @@
 #include <torch/script.h>
 
 using c10::Dict;
+using c10::IValue;
 
 namespace torchtext {
 namespace {
 
+typedef std::tuple<std::string, std::vector<int64_t>, std::vector<std::string>,
+                   std::vector<torch::Tensor>>
+    VectorsStates;
+
 struct Vectors : torch::CustomClassHolder {
 public:
+  const std::string version_str_ = "0.0.1";
```

Comment:
I think you can make this a constexpr, if you want:  https://stackoverflow.com/questions/46100310/how-to-declare-constexpr-c-string

Reply:
Got it. I wasn't fully aware of what the difference btw `const` and `consexpr` was but from what I understand, a constexpr variable must be initialized at compile time rather than run time which should help the program run faster and use less memory.

Reply:
Not going to pursue this as per our offline conversation. Using `constexpr` leads to needing to use literals (i.e. `char[]`). This means we need to do conversions to std::string anyways which isn't worth the potential perf gains.